commandapi-nms/commandapi-1.19-common/src/main/java/dev/jorel/commandapi/nms/NMS_1_19_Common.java

```diff
@@ -160,6 +160,8 @@
 import net.minecraft.world.level.gameevent.EntityPositionSource;
 import net.minecraft.world.phys.Vec2;
 import net.minecraft.world.phys.Vec3;
+import org.jetbrains.annotations.Contract;
+import org.jetbrains.annotations.NotNull;
```

Comment:
Unnecessary imports that aren't useful for anyone other than maintainers. The methods these are used on are only being used internally by the CommandAPI and aren't publicly seen by end users (main developers), so they have no main purpose.

In general, almost all methods in the CommandAPI are non-null, so these are effectively obsolete here.